drivers/dummy/texture_loader_dummy.cpp

```diff
@@ -45,10 +45,6 @@ RES ResourceFormatDummyTexture::load(const String &p_path, const String &p_origi
 
 	dstbuff.resize(rowsize * height);
 
-	PoolVector<uint8_t>::Write dstbuff_write = dstbuff.write();
```

Comment:
Seems OK to remove, it was copied over from `drivers/png/image_loader_png.cpp` but `data` is not used here.